sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/DataFeedLiveTests.cs

```diff
@@ -2070,61 +2069,85 @@ public async Task DeleteDataFeed(bool useTokenCredential)
                 Description = "This data feed was created to test the .NET client.",
                 AccessMode = DataFeedAccessMode.Public,
                 ActionLinkTemplate = "https://fakeurl.com/%metric/%datafeed",
-                Administrators = new List<string>() { "fake@admin.com" },
-                Viewers = new List<string>() { "fake@viewer.com" },
                 MissingDataPointFillSettings = new () { FillType = DataFeedMissingDataPointFillType.CustomValue, CustomFillValue = 45.0 }
             };
+
+            dataFeed.Administrators.Add("fake@admin.com");
+            dataFeed.Viewers.Add("fake@viewer.com");
+
+            dataFeed.Schema.MetricColumns.Add(new ("cost") { MetricDisplayName = "costDisplayName", MetricDescription = "costDescription" });
+            dataFeed.Schema.MetricColumns.Add(new ("revenue") { MetricDisplayName = "revenueDisplayName", MetricDescription = "revenueDescription" });
+
+            dataFeed.Schema.DimensionColumns.Add(new ("city"));
+            dataFeed.Schema.DimensionColumns.Add(new ("category") { DimensionDisplayName = "categoryDisplayName" });
+
+            return dataFeed;
         }
 
-        private void SetOptionalMembers(DataFeed dataFeed)
+        private void SetOptionalMembers(DataFeed dataFeed, string dataFeedName)
         {
+            dataFeed.Name = dataFeedName;
             dataFeed.Description = "This data feed was updated to test the .NET client.";
             dataFeed.AccessMode = DataFeedAccessMode.Public;
             dataFeed.ActionLinkTemplate = "https://fakeurl.com/%datafeed/%metric";
-            // TODO: add administrator update validation. Related: https://github.com/Azure/azure-sdk-for-net/issues/17766
+
+            // - If we're creating the data feed from scratch, currently the Administrators list has no elements.
+            //   If we add a fake admin and send it to the service, it will overwrite the current list of admins,
+            //   removing ourselves from the list. Doing so would cause permission errors during the next service
+            //   calls.
+            // - If we're updating a data feed obtained from a GetDataFeed operation, the Administrators list already
+            //   includes our email. Even if we add new admins, this will not remove our admin role.
+            //
+            // For this reason, we do a conditional validation in the ValidateUpdatedDataFeedWithOptionalMembersSet
+            // method.
+
+            if (dataFeed.Administrators.Count > 0)
+            {
+                dataFeed.Administrators.Add("fake@admin.com");
+            }
+
             dataFeed.Viewers.Add("fake@viewer.com");
 
+            dataFeed.Schema = new DataFeedSchema();
             dataFeed.Schema.TimestampColumn = "updatedTimestampColumn";
 
+            dataFeed.IngestionSettings = new DataFeedIngestionSettings();
+            dataFeed.IngestionSettings.IngestionStartTime = DateTimeOffset.Parse("2020-09-21T00:00:00Z");
```

Comment:
Since we can freely set the ingestion start time now as well, adding validation in the update method here.